lib/basedriver/commands/images.js

```diff
@@ -0,0 +1,29 @@
+import _ from 'lodash';
+import { errors } from '../../protocol/errors';
+import { imageUtil } from 'appium-support';
```

Comment:
do we need to update package.json to ensure `appium-support` is at a version which has this library method?

Reply:
yep, I'm waiting until the module is published